### PR TITLE
ReadMe Review

### PR DESCRIPTION
**Pull request by helenktsai**
- [x] Apply format template with link to component documentation
- [x] Ensure no dead links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,16 @@
 # NServiceBus.AmazonSQS
 
-This is an Amazon SQS transport for NServiceBus.
+NServiceBus.AmazonSQS is the Amazon SQS transport for NServiceBus.
```
Comment (soujay) [inline README.md:3]:
```suggestion
NServiceBus.AmazonSQS is the [Amazon SQS](https://docs.aws.amazon.com/sqs/) transport for NServiceBus.
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,16 @@
...
-* **NSERVICEBUS_AMAZONSQS_S3BUCKET** corresponds to the [S3BucketForLargeMessages](https://docs.particular.net/transports/sqs/configuration-options#s3bucketforlargemessages) parameter. Default is no S3 bucket.
-
-The bucket should not have encryption enabled. An additional bucket `{NSERVICEBUS_AMAZONSQS_S3BUCKET}.kms` with AWS KMS encryption enabled is required.
-
-Additional environment variables required for AWS:
-
- * **AWS_ACCESS_KEY_ID** access key ID to sign programmatic requests that you make to AWS. Provisioned via IAM.
- * **AWS_SECRET_ACCESS_KEY** secret access key to sign programmatic requests that you make to AWS. Provisioned via IAM.
- * **AWS_REGION** Valid AWS region.
+To run the tests, the Access Key ID and Secret Access Key of an AWS IAM account need to be set in environment variables on the machine running the tests. Full details on how to set this up can be found in the [Amazon SQS Transport documentation](https://docs.particular.net/transports/sqs/).
```
Comment (soujay) [inline README.md:13]:
```suggestion
To run the tests, the Access Key ID and Secret Access Key of an AWS IAM account need to be set in environment variables on the machine running the tests. Full details on how to set this up can be found in the [Amazon SQS Transport documentation](https://docs.particular.net/transports/sqs/#prerequisites).
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,16 @@
...
+## Running tests locally
 
 The solution contains the [NServiceBus Acceptance Test suite](https://www.nuget.org/packages/NServiceBus.AcceptanceTests.Sources/) and the [NServiceBus Transport Test suite](https://www.nuget.org/packages/NServiceBus.TransportTests.Sources/).
-To run the tests, the Access Key ID and Secret Access Key of an AWS IAM account need to be set in environment variables on the machine running the tests. Full details on how to set this up can be found [here](https://docs.particular.net/transports/sqs/#getting-started-set-up-an-aws-account).
 
-The transport can be configured using the following environment variables:
-
-* **NSERVICEBUS_AMAZONSQS_S3BUCKET** corresponds to the [S3BucketForLargeMessages](https://docs.particular.net/transports/sqs/configuration-options#s3bucketforlargemessages) parameter. Default is no S3 bucket.
-
-The bucket should not have encryption enabled. An additional bucket `{NSERVICEBUS_AMAZONSQS_S3BUCKET}.kms` with AWS KMS encryption enabled is required.
```
Comment (soujay) [inline README.md:18]: I think it is best to add this back in as the variable is used in ci.yml 
* **NSERVICEBUS_AMAZONSQS_S3BUCKET** corresponds to the [S3BucketForLargeMessages](https://docs.particular.net/transports/sqs/configuration-options#offload-large-messages-to-s3) parameter. By default this is disabled. The bucket should not have encryption enabled. An additional bucket `{NSERVICEBUS_AMAZONSQS_S3BUCKET}.kms` with AWS KMS encryption enabled is required. For further details review the documentation on [configuration options](https://docs.particular.net/transports/sqs/configuration-options).

File: README.md [hunk 4]
```diff
@@ -1,27 +1,16 @@
...
```
Comment (helenktsai) [inline README.md:18]: The transport can be configured using the following environment variables:

* **NSERVICEBUS_AMAZONSQS_S3BUCKET** corresponds to the [S3BucketForLargeMessages](https://docs.particular.net/transports/sqs/configuration-options#s3bucketforlargemessages) parameter. Default is no S3 bucket.

The bucket should not have encryption enabled. An additional bucket `{NSERVICEBUS_AMAZONSQS_S3BUCKET}.kms` with AWS KMS encryption enabled is required.

**Review by soujay — APPROVED** [review]

**Pull request merged**